src/lib/components/DeckGLMap/layers/wells/wellsLayer.ts

```diff
@@ -8,6 +8,7 @@ export interface WellsLayerProps<D> extends CompositeLayerProps<D> {
     pointRadiusScale: number;
     lineWidthScale: number;
     outline: boolean;
+    selectedFeature: string;
```

Comment:
I think this is a GeoJSON feature object, not a string. I've added a package with geojson types, you can see it used in the drawing layer. For this case, you can probably use the `Feature` type from there. Also, you can replace the `WellDataType` with a type from that package (maybe FeatureCollection?), and just create a type for the custom properties you've added to the geojson.